### PR TITLE
Modify HDF5 class

### PR DESCRIPTION
**Pull request by k2kobayashi**
I implemented follows:
- mainly using 'a' option instead of using 'w' option
- Implement additional writing of one h5 file
- change example script

### CODE REVIEW

File: sprocket/util/tests/test_hdf5.py [hunk 1]
```diff
@@ -28,9 +29,14 @@ def test_HDF5(self):
...
         tmph5.close()
-
         assert np.allclose(tmp1d, data1d)
         assert np.allclose(tmp2d, data2d)
 
+        # read/write and replace test
+        h5 = HDF5(path, 'a')
+        tmp1d = h5.read(ext='1d')
+        h5.save(data1d2, '1d')
+        h5.close()
```
Comment (tats-u) [inline sprocket/util/tests/test_hdf5.py]: I suppose some `assert` should be added.

Comment (k2kobayashi) [reply sprocket/util/tests/test_hdf5.py]: Thanks. I add them.

**Comment by tats-u** [comment]
👌 to be merged

**Comment by k2kobayashi** [comment]
Thanks @tats-u .

**Pull request merged**